acbuild/write.go

```diff
@@ -33,7 +34,8 @@ var (
 func init() {
 	cmdAcbuild.AddCommand(cmdWrite)
 
-	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite output ACI")
+	cmdWrite.Flags().BoolVar(&overwrite, "overwrite", false, "overwrite the resulting ACI")
+	cmdWrite.Flags().BoolVar(&compress, "compress", false, "compress the resulting ACI")
```

Comment:
If it compresses by default, then should I make the option `--skip-compression`?

In that case, then you'd either be doing `--compress=false` or `--skip-compression`.

Is not compressing really a need?